rules/do-you-have-a-knowledge-base-kb/rule.md

```diff
@@ -26,54 +26,42 @@ So, if you answer questions on your products to customers, you are wasting time
 
 Now of course there are many customers who don't look for a KB, but instead, you fire off the same old email that you already know is an MDAC related error, and your current solution is to tell them to run SSW Diagnostics and get all the green ticks.
 
-
 ::: greybox
 
-**Dear Harry,**
-
-Thank you for taking the time to report the issue to SSW Code Auditor. I'm happy to let you know that this is a known issue and has been addressed in our knowledge base: [link]
+**Dear Harry,**  
 
+Thank you for taking the time to report the issue to SSW CodeAuditor. I'm happy to let you know that this is a known issue and has been addressed in our knowledge base: {{ LINK }}
 
-
-
-Thanks,
+Thanks,  
 Bob
-
-
 :::
-
-
 ::: good
 Figure: Responding to a known issue with a KB article  
 :::
 
 The basic rule is: don't send back the answer in your email - instead send back the link. More specifically:
 
-1. If you can answer a support email then reply to the support email (using one of the 4 templates on the right)
-    * TO: the client
-    * CC: the developer and your manager with the KB link
-2. If you can’t answer the question then reply to the support email
-    * TO: the client and the developer
-    * CC: your manager
+1. **If you can answer** a support email then reply to the support email
+    * To: the client
+    * Cc: the developer and your manager with the KB link
+2. **If you can’t answer** the question then reply to the support email:
+    * To: the client and the developer
+    * Cc: your manager
     * Ask the customer if they can get diagnostics to all green ticks.
     * Ask the developer to “Please action?"
 
-
-
 ::: greybox
 
 **Dear Harry,**
 
-Thank you for taking the time to report the issue to SSW Code Auditor.
+Thank you for taking the time to report the issue to SSW CodeAuditor.
 
-I am sorry to let you know that I cannot reproduce this. Could you please provide me with more details or, even better, would I be able to connect to your PC? It is simple and you can see everything I do. To do so, you can send me an appointment for an appropriate time or add me to your Skype
+I am sorry to let you know that I cannot reproduce this. Could you please provide me with more details or, even better, would I be able to connect to your PC? It is simple and you can see everything I do. To do so, you can send me an appointment for an appropriate time or add me to Skype
```

Comment:
Bit unusual nowadays to allow anyone to "Connect to your PC".

Instead, the support person can ask Harry if its ok for them to set up a Teams call (or any other video call) where Harry can share screens and show the support person the issue.